scripts/remove_idle_group_members.pl

```diff
@@ -37,7 +37,7 @@
   q{SELECT DISTINCT profiles.userid AS user_id,
             groups.id AS group_id
        FROM profiles JOIN user_group_map ON profiles.userid = user_group_map.user_id
-            JOIN groups ON user_group_map.group_id = groups.id
+            JOIN `groups` ON user_group_map.group_id = groups.id
```

Comment:
Instead of using ` quotes directly we use a helper function called $dbh->quote_identifier('groups') in case another DB other than MySQL uses different type of quotes for the same thing. So in this case it would be:

```perl
JOIN ' .  $dbh->quote_identifier('groups') . ' ON user_group_map.group_id = groups.id
```

I can fix this if you want. It was missed somehow when I had to do the same change in many other places in the code.

Reply:
Sure, that would be great!